### PR TITLE
tests: On ExecuteCommandOnPod failure print both err and stderr is exist

### PR DESCRIPTION
**Pull request by AlonaKaplan**
**What this PR does / why we need it**:

On ExecuteCommandOnPod failure print both err and stderr is exist.
Sometimes stderr contains more information than the err.

For example, I encountered the following -
"**err**: command terminated with exit code 1, **stderr**: cat: /sys/class/net/net12/mtu: No such file or directory"

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/utils.go [hunk 1]
```diff
@@ -3135,6 +3135,10 @@ func NewRepeatableVirtctlCommand(args ...string) func() error {
 func ExecuteCommandOnPod(virtCli kubecli.KubevirtClient, pod *k8sv1.Pod, containerName string, command []string) (string, error) {
 	stdout, stderr, err := ExecuteCommandOnPodV2(virtCli, pod, containerName, command)
 
+	if err != nil && len(stderr) > 0 {
+		return "", fmt.Errorf("err: %v, stderr: %v", err, stderr)
```
Comment (qinqon) [inline tests/utils.go:3139]: Normally at go we nest the errors with `:` so they are like "error 1: error 2: error 3", let's trait stderr like another error and add "error" at the end.
```golang
fmt.Errorf("failed executing command  on pod: %v: %v", stderr, err)
```

**Review by qinqon — CHANGES_REQUESTED** [review]

File: tests/utils.go (hunk 1 above)
Comment (AlonaKaplan) [reply tests/utils.go:3139]: And keep the other failures in this method as is?

Comment (qinqon) [reply tests/utils.go:3139]: Think it about it again, I think we have to print stdout too is good for debugging and in this case is not a bad thing to state taht we are pringing "stderr" so what I mean is the following 

```golang
if err != nil {
   err = fmt.Errorf("failed executing command on pod: %v", err)
   if len(stderr) > 0 {
		err = fmt.Errorf("%v: %s stderr: %s: stdout: %s", err, stdout, stderr)
   }
   return err
}
```

Comment (AlonaKaplan) [reply tests/utils.go]: done. In case of an error I'm printing `stderr` and `stdout` even if they are empty.

**Review by enp0s3 — APPROVED** [review]
Thank you!
/approve

**Pull request merged**